AK/StringUtils.h

```diff
@@ -9,6 +9,7 @@
 
 #include <AK/Concepts.h>
 #include <AK/Forward.h>
+#include <AK/Function.h>
```

Comment:
No need to add this `#include` here. The forward declaration in `AK/Forward.h` should take care of it.